tools/create-openbsd-gce-ci.sh

```diff
@@ -29,9 +29,8 @@ fi
 mkdir -p etc
 cat >install.site <<EOF
 #!/bin/sh
-set -x
 syspatch
-pkg_add -iv bash git go
+pkg_add -iv bash git gmake go && echo pkg_add OK
```

Comment:
please also add wget to download images/etc and nano as a simple text editor

Reply:
also need clang-format, can't figure out the right package

Reply:
Apparently it's in llvm package, so the image is gonna be humongous:
https://github.com/openbsd/ports/search?q=clang-format&unscoped_q=clang-format

I wonder if I can convince people to create a subpackage...